common/app/conf/switches.scala

```diff
@@ -789,6 +788,15 @@ object Switches {
     exposeClientSide = false
   )
 
+  val IdentityCookieRefreshSwitch = Switch(
```

Comment:
worth considering deleting the switch altogether if it's not something we'll want to tweak in future